src/clj/chronograph/db/user.clj

```diff
@@ -35,3 +35,11 @@
    (first (sql/query tx
                      ["SELECT * FROM users WHERE id=?" user-id]
                      db/sql-opts))))
+
+(defn find-by-org-id
+  ([tx organization-id]
+   (sql/query tx
+              ["SELECT users.* FROM users, acls
+                WHERE users.id = acls.user_id 
+                AND acls.organization_id=?" organization-id]
```

Comment:
I _think_ this implicitly is an inner join. Correct? If so, does it make more sense to explicitly write it as an inner join? I read on the Internet that explicit is better than implicit in such cases, for query planning (and maybe performance) reasons.

Reply:
This is an inner join. I don't think this syntax adversely impacts performance, at least as far as postgres is concerned. In fact, this syntax is recommended in the official tutorial.